tests/stress/km/stress_tests_km.cpp

```diff
@@ -210,6 +197,44 @@ _restart_extension(const std::string& extension_name, uint32_t timeout)
     return status;
 }
 
+static std::thread
+_start_extension_restart_thread(const std::string& extension_name, uint32_t restart_delay, uint32_t thread_lifetime)
```

Comment:
recommend changing variable names to make the units clear.  Is `restart_delay` in seconds?  ms?
What about thread_lifetime?  I suspect it should be restart_delay_ms and thread_lifetime_minutes

Reply:
fixed